.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.js linguist-generated
```

Comment:
can we add an exclusion.  jest.config.js is the only one that I can think of ...